drivers/soc/atmel/soc.c

```diff
@@ -70,11 +70,11 @@ static const struct at91_soc __initconst socs[] = {
 #ifdef CONFIG_SOC_SAM9X60
 	AT91_SOC(SAM9X60_CIDR_MATCH, SAM9X60_EXID_MATCH, "sam9x60", "sam9x60"),
 	AT91_SOC(SAM9X60_CIDR_MATCH, SAM9X60D5M_EXID_MATCH,
-		 "sam9x60d6m", "sam9x60"),
+		 "sam9x60d5m 512MiB DDR2 SiP", "sam9x60"),
 	AT91_SOC(SAM9X60_CIDR_MATCH, SAM9X60D1G_EXID_MATCH,
-		 "sam9x60d1g", "sam9x60"),
+		 "sam9x60d1g 1GiB DDR2 SiP", "sam9x60"),
```

Comment:
Well, no, this is not 1GiB, it's 128MiB: please correct all them.

Reply:
Bits and bytes - always difficult. I will fix it.